src/test/java/com/google/javascript/clutz/partial/legacy_namespace_forward_declare.d.ts

```diff
@@ -0,0 +1,24 @@
+declare namespace ಠ_ಠ.clutz.legacy.namespace.forward {
+  type declare = ಠ_ಠ.clutz.module$contents$legacy$namespace$forward$declare_ClassExtendingMissingForwardDeclare ;
+  var declare : typeof ಠ_ಠ.clutz.module$contents$legacy$namespace$forward$declare_ClassExtendingMissingForwardDeclare ;
+  type declare_Instance = ಠ_ಠ.clutz.module$contents$legacy$namespace$forward$declare_ClassExtendingMissingForwardDeclare_Instance ;
+  var declare_Instance : typeof ಠ_ಠ.clutz.module$contents$legacy$namespace$forward$declare_ClassExtendingMissingForwardDeclare_Instance ;
+}
+declare module 'goog:legacy.namespace.forward.declare' {
+  import alias = ಠ_ಠ.clutz.legacy.namespace.forward.declare;
+  export default alias;
+}
+declare namespace ಠ_ಠ.clutz {
+  class module$contents$legacy$namespace$forward$declare_ClassExtendingMissingForwardDeclare extends module$contents$legacy$namespace$forward$declare_ClassExtendingMissingForwardDeclare_Instance {
+  }
+//!! Without including forward declares as imports, ಠ_ಠ.clutz.module$exports$missing$forward$Declare_Instance would be ಠ_ಠ.clutz.module$contents$missing$forward$Declare
+  class module$contents$legacy$namespace$forward$declare_ClassExtendingMissingForwardDeclare_Instance extends ಠ_ಠ.clutz.module$exports$missing$forward$Declare_Instance {
+    constructor ( ) ;
+  }
+}
+declare namespace ಠ_ಠ.clutz {
```

Comment:
I meant add //!! for this block, something like "//!! Upstream consumers expect goog.module-shaped import, but we have generated goog.provide-shaped. This block links the two."

But maybe that is too obvious?